api/core/tools/provider/builtin/comfyui/comfyui.yaml

```diff
@@ -0,0 +1,26 @@
+identity:
+  author: Dify
+  name: comfyui
+  label:
+    en_US: ComfyUI
+    zh_Hans: ComfyUI
+    pt_BR: ComfyUI
+  description:
+    en_US: ComfyUI is a tool for generating images which can be deployed locally.
+    zh_Hans: ComfyUI 是一个可以在本地部署的图片生成的工具。
+    pt_BR: ComfyUI is a tool for generating images which can be deployed locally.
+  icon: icon.png
+  tags:
+    - image
+credentials_for_provider:
+  base_url:
+    type: secret-input
```

Comment:
This does not need to be a `secret-input`.

Reply:
I has used `text-input` instead of `secret-input`.